### ISSUE TITLE
[Keyboard Navigation - Bot Webchat - Suggested actions ]: Keyboard focus indicator is not clearly visible on card control.

### ISSUE THREAD

**Issue by KattaBhargav** [microsoft/BotFramework-WebChat#4029]
User Experience:
If focus indicator is not clearly visible on card controls, then users with motor disabilities who depend on keyboard for navigation will not be able to identify the focus on that control which can prevent them from performing actions which are important.

Repro Steps:
1. Open URL: Web Chat: Full-featured bundle (microsoft.github.io)
2. Web chat bundle will open.
3. Navigate on the web chat bot and enter "suggested-actions’" keyword in Type your message box, then select send button.
4. Cards related to the keyword will appear, navigate on the cards.
5. Check and verify that step 3-4 are accessible and MAS compliant.

Actual Result:
Keyboard focus indicator is not clearly visible on card control.

Expected Result:
Keyboard focus indicator should be clearly visible on card control.

MAS Reference:
WCAG2.4.7 – Focus Visible

Test Environment:
OS: Win 10 21H1
Build: OS Build 19043.1147
Browser: Edge Chromium Version 94.0.972.2 (Official build) dev (64-bit)
#URL: Web Chat: Full-featured bundle (microsoft.github.io)

**Comment by KattaBhargav** [comment]
#A11yMAS;#A11ySev2;#HCL;#MAS2.4.7;#Benchmark;#Accessibility;#Win10-Edge(Chromium);#E2E-HCL-BotFrameworkWebChat-Aug21;

**Comment by compulim** [comment]
We will update our style to align with [Fluent UI Default Button](https://developer.microsoft.com/en-us/fluentui#/controls/web/button). Particularly:

- Single pixel border (currently, it is double pixel border)
- Very slight roundness (possibly, 1px roundness)
- Double border when `:focus-visible` (focus by keyboard)
- Light-dim on mouse hover
- Dim on click via keyboard or mouse

**Comment by compulim** [comment]
![image](https://user-images.githubusercontent.com/1622400/133176381-b8ae97bc-dc4e-44fa-99be-6c125a3f102f.png)

This is the style for the button.

![image](https://user-images.githubusercontent.com/1622400/133176582-98841b68-4b19-4ce0-acfd-445ece37ddf1.png)

This is the style when :focus-visible is applied to the button.

![image](https://user-images.githubusercontent.com/1622400/133176647-a9051da4-273b-450a-86f2-4f8f417d12de.png)

**Comment by compulim** [comment]
I am keeping this open but resolved.

**Comment by compulim** [comment]
The issue is fixed in #4035.

Here is a quick recap of how it works.

![recording (28)](https://user-images.githubusercontent.com/1622400/134995080-6b734c92-20cc-49bb-a7ea-519ecd159973.gif)

**Comment by v-shikch** [comment]
As verified issue on URL: Web Chat: Full-featured bundle (microsoft.github.io) on Browser: Edge Chromium Version 96.0.1028.0 (Official build) dev (64-bit), here issue is fixed. Hence closing the bug.

**Comment by compulim** [comment]
Thanks @v-shikch and @KattaBhargav. Appreciated.
